client/app/visualizations/choropleth/Editor/ColorsSettings.jsx

```diff
@@ -101,6 +106,7 @@ export default function ColorsSettings({ options, onOptionsChange }) {
         <Grid.Col span={12}>
           <ColorPicker
             id="choropleth-editor-color-background"
+            data-test="Choropleth.Editor.Colors.Backbround"
```

Comment:
:slightly_smiling_face: 

```suggestion
            data-test="Choropleth.Editor.Colors.Background"
```

Reply:
:man_facepalming: 

Thank you @gabrieldutra!